src/app/modules/workers/components/delete-worker/delete-worker.component.html

```diff
@@ -0,0 +1,3 @@
+<button mat-raised-button class="button" (click)="onSubmit()">
```

Comment:
Uwaga dotyczy funkcjonalnosci add/edit/delete

w tych rozwiązaniach jest silne sprzężenie między przyciskiem i formularzem/dialogiem, czyli nie możemy otworzyc dialogu inaczej niz poprzez ten przycisk, tak samo nie da się np. przekierować użytkownika na dany dialog. Lepiej rodzielić te dwa elementy i pozowolic wykorzystywac nasze dialogi/formularze w elastyczny sposob

dialog nie powinien wiedzieć w jaki sposób jest otwierany

ograniczajmy też do minimum miejsca które mają zależność od fasad

Reply:
co do zależności od fasad, to właśnie zastanawiałem sie czy w edit/delete wynosić te eventy w górę do workers page gdzie jest już fasada, bo trzeba by to przekazywać w górę aż trzy razy